docs/src/main/tut/chisel3/operators.md

```diff
@@ -51,3 +51,14 @@ Chisel defines a set of hardware operators:
 We have to use triple equals```===``` for equality and ```=/=```
 for inequality to allow the
 native Scala equals operator to remain usable.
+
+The Chisel operator precedence is determined by the evaluation order of the circuit,
+which follows the [Scala operator precedence](https://docs.scala-lang.org/tour/operators.html).
+If in doubt, it is always a good praxis to use parentheses.
```

Comment:
`"praxis"` sounds a bit antiquated, but it's technically correct. As a nit, `"practice"` would sound better.